Britannia/common/scripted_triggers/BTWK_storyline_triggers.txt

```diff
@@ -0,0 +1,25 @@
+﻿holds_any_storyline_britannia_kingdoms = {
```

Comment:
Maybe rename to hold_kingdom_near_dumnonia or something? Current name doesn't really properly describe what this is supposed to be IMO.